plugins/generic/externalFeed/ExternalFeedBlockPlugin.inc.php

```diff
@@ -73,6 +72,14 @@ function getPluginPath() {
 		return $plugin->getPluginPath();
 	}
 
+	/**
+	 * Override the builtin to get the correct template path.
+	 * @return string
+	 */
+	function getTemplatePath() {
```

Comment:
@NateWr has recently added some changes that might require work here as part of https://github.com/pkp/pkp-lib/issues/2872.